### PR TITLE
Bypass CI issues for rethinkdb

### PR DESCRIPTION
**Pull request by kansi**

### CODE REVIEW

File: tests/integration/test_federation.py [hunk 1]
```diff
@@ -7,6 +7,7 @@
 from contextlib import contextmanager
 from bigchaindb.common.crypto import generate_key_pair
 from tests.pipelines.stepping import create_stepper
+from tests.conftest import skip_travis_rdb
```
Comment (sbellem) [inline tests/integration/test_federation.py]: Is this import really needed?

Comment (kansi) [reply tests/integration/test_federation.py]: Is there another way I can access `skip_travis_rdb` in this module?

Comment (kansi) [reply tests/integration/test_federation.py]: @sbellem I think I can directly do this as `@pytest.mark.usefixtures("skip_travis_rdb")`

**Comment by sbellem** [comment]
**NOTE**: The approach used was based on https://docs.pytest.org/en/latest/example/markers.html#marking-platform-specific-tests-with-pytest

**Pull request merged**